gamefixes-steam/2677660.py

```diff
@@ -0,0 +1,14 @@
+"""Game fix for Indiana Jones and the Great Circle"""
+
+from protonfixes import util
+
+
+def main() -> None:
+	"""Workarounds for various graphics driver bugs encountered in this game"""
+	# RadV: Missing textures and glowing eyes
```

Comment:
Skimming through the relevant upstream Mesa issue, [the graphics-related issue when using RADV seems to have already been resolved for this game](https://gitlab.freedesktop.org/mesa/mesa/-/issues/12257). Remove the 3 lines below.

cc @matte-schwartz for confirmation

Reply:
correct, and they will all be in the upcoming Mesa stable release this week